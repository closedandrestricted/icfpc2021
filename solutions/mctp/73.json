{"vertices":[[15,60],[18,55],[5,62],[13,52],[7,53],[19,38],[28,38],[26,38],[20,46],[26,43],[24,38],[7,62],[28,44],[21,38],[21,40],[15,64],[21,39],[26,41],[7,72],[22,63],[15,40],[5,68],[25,33],[31,54],[44,6],[42,6],[39,16],[42,42],[41,9],[47,16],[47,18],[38,12],[38,10],[44,15],[37,16],[42,22],[33,16],[31,37],[24,30]]}