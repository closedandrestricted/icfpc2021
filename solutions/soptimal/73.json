{"vertices":[[51,0],[56,1],[43,5],[31,12],[49,11],[18,24],[10,25],[12,25],[26,21],[15,32],[11,29],[49,20],[9,31],[6,47],[6,49],[48,27],[12,39],[7,41],[59,20],[41,30],[6,55],[55,18],[15,46],[34,41],[0,76],[2,76],[4,65],[28,56],[1,72],[5,77],[5,75],[4,69],[2,70],[2,76],[8,71],[11,76],[7,67],[29,68],[23,77]]}