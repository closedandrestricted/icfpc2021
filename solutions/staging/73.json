{"vertices":[[31,14],[34,18],[28,23],[13,25],[36,29],[16,41],[11,56],[13,56],[22,30],[17,49],[13,52],[34,38],[10,63],[20,56],[22,56],[38,45],[15,56],[17,61],[25,38],[32,42],[26,61],[29,39],[15,63],[28,30],[34,52],[36,52],[26,44],[42,24],[34,48],[37,49],[37,47],[30,46],[40,48],[40,50],[34,47],[38,53],[35,43],[50,16],[46,27]]}