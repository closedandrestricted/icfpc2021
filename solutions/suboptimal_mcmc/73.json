{"vertices":[[51,0],[56,1],[43,6],[31,12],[51,11],[21,27],[12,25],[10,25],[36,19],[16,34],[14,29],[48,20],[12,32],[19,38],[19,40],[47,28],[13,40],[17,43],[58,20],[42,33],[22,35],[54,19],[17,45],[40,44],[4,72],[4,74],[10,66],[34,58],[8,74],[2,73],[4,73],[10,70],[4,68],[1,76],[7,73],[4,67],[6,77],[29,68],[23,77]]}