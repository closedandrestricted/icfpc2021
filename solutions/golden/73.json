{"vertices":[[51,0],[56,1],[43,5],[31,12],[49,11],[21,25],[10,25],[12,25],[32,21],[15,31],[14,26],[49,20],[9,31],[15,39],[15,37],[50,28],[14,38],[9,39],[59,20],[43,31],[9,35],[55,18],[16,45],[40,43],[0,76],[2,76],[8,66],[33,58],[3,73],[5,77],[7,77],[6,70],[7,73],[8,76],[2,71],[13,77],[6,70],[29,68],[23,77]]}